components/autofill/android/lint-baseline.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 8.12.1" type="baseline" client="gradle" dependencies="false" name="AGP (8.12.1)" variant="all" version="8.12.1">
+
+    <issue
```

Comment:
This worries me, but surely I'm missing something.  Is this lint saying that we're going to regress our 16kb alignment (needed in November, IIRC) by continuing to use JNA 5.14.0?

Reply:
Generally, I'd rather see us disable this lint in these projects rather than boil all of this version specific nonsense into these files.  Is that possible?

Reply:
sgtm, I just assumed this was somehow "normal" - I'll work out how to disable the lints.